rosetta-source/src/main/rosetta/event-common-type.rosetta

```diff
@@ -103,6 +103,8 @@ type ExecutionInstruction: <"Specifies instructions for execution of a transacti
 	executionDetails ExecutionDetails (1..1) <"Specifies the type and venue of execution, e.g. via voice, or electronically.">
 	tradeDate date (1..1) <"Denotes the trade/execution date.">
 	    [metadata id]
+	tradeDateTime zonedDateTime (0..1) <"Denotes the trade execution date and time with timezone.">
```

Comment:
As per my comment on the associated Issue I think we should have separate trade date and trade time/timezone items.